array.c

```diff
@@ -6144,8 +6134,6 @@ rb_ary_flatten_bang(int argc, VALUE *argv, VALUE ary)
  *  - Each non-Array element is unchanged.
  *  - Each \Array is replaced by its individual elements.
  *
- *  Argument +level+, if given and not +nil+, must be Integers.
- *
  *  With non-negative argument +level+, flattens recursively through +level+ levels:
```

Comment:
Not a big deal, but you may want to use `non-negative \Integer argument` as you did above.

Reply:
Fixed.